UnitedIncome/components
# Pie color changes and exposing StepProps

### Description
> Please provide a brief description of your changes below.

Accounting for painting `Municipal Bonds` in `client` `double-donut`s requires updating our pie color palette. I coordinated with James/Monique to update these colors.

Also, I created two arrays we can reference for changing the default color of a story background.

Lastly, `StepProps` is referenced in `client` and should be exposed so that we can directly reference this type from component library.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Only the background color change is testable.

1. `yarn start` this branch locally
2. Ensure that when you go to the `chanceOfSuccess` story your background color changes

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

Example of background color changing dynamically
[![Example of background color changing dynamically](https://i.gyazo.com/e3b254ba636d09aa1cb02088862aa876.gif)](https://gyazo.com/e3b254ba636d09aa1cb02088862aa876)

### Comments
> Any additional comments that will aid with the review process.

This ticket blocks `app-4820`.

- [Related client Ticket](https://unitedincome.atlassian.net/browse/APP-4820)
